staging/src/k8s.io/component-base/logs/json/register/register.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package register
+
+import (
+	"k8s.io/component-base/logs"
+	json "k8s.io/component-base/logs/json"
+)
+
+func init() {
+	// JSON format is optional klog format
```

Comment:
Can you check if registering json twice will work and not result in panic?
.We will only be able to remove original registration when all modules will be migrated. During migration we will be calling register twice.

Reply:
I have tested that double registration. It does not cause panic and will work.
Because this method call does not catch the error returned.https://github.com/kubernetes/kubernetes/blob/f7cff077d5693d2c5e199aa18c002a4c25667f4c/staging/src/k8s.io/component-base/logs/config.go#L42 


https://github.com/kubernetes/kubernetes/blob/f7cff077d5693d2c5e199aa18c002a4c25667f4c/staging/src/k8s.io/component-base/logs/registry.go#L45-L47

Therefore, there is still only one `json` key for the map filed `registry` of package-level variable `LogRegistry`
https://github.com/kubernetes/kubernetes/blob/f7cff077d5693d2c5e199aa18c002a4c25667f4c/staging/src/k8s.io/component-base/logs/config.go#L37